server/routes/v1/validators.js

```diff
@@ -1,49 +1,29 @@
 const axios = require('axios');
 const log = require('../../lib/logger')({ name: 'validators' });
 
+const ENV_NETWORK_MAP = {
+  mainnet: 'main',
+  testnet: 'test',
+  devnet: 'dev',
+  'nft-devnet': 'nft-dev',
+};
+
 const cache = {};
 
-const fetchValidators = () =>
-  axios
-    .get(`${process.env.REACT_APP_DATA_URL}/validators`)
+const fetchValidators = () => {
+  const network = ENV_NETWORK_MAP[process.env.REACT_APP_ENVIRONMENT];
+  return axios
+    .get(`${process.env.REACT_APP_DATA_URL}/validators/${network}`)
     .then(response => response.data.validators);
-
-const fetchDomains = () =>
-  axios.get('https://data.ripple.com/v2/network/validators').then(response => {
-    response.data.validators.forEach(d => {
-      if (d.domain) {
-        const validator = cache.validators.find(v => v.master_key === d.validation_public_key);
-        if (validator) {
-          if (!(validator.domain && validator.domain_is_verified)) {
-            validator.domain = d.domain;
-          }
-        }
-      }
-    });
-  });
+};
 
 const cacheValidators = async () => {
   if (!cache.pending) {
     cache.pending = true;
     try {
       cache.validators = await fetchValidators();
-      // Testnet/Devnet validators do not need a 30day score.
-      if (
-        process.env.REACT_APP_ENVIRONMENT === 'testnet' ||
-        process.env.REACT_APP_ENVIRONMENT === 'devnet'
-      ) {
-        cache.validators = cache.validators.filter(v => v.unl === process.env.REACT_APP_VALIDATOR);
-      } else {
-        cache.validators = cache.validators.filter(
-          v =>
-            v.agreement_30day &&
-            v.agreement_30day.score &&
-            (v.unl === process.env.REACT_APP_VALIDATOR || v.unl === false)
-        );
-      }
```

Comment:
Is the filtering now happening on the api side so we don't need to manually check here? 
(In other words, why was this deleted?)

Reply:
Yes - see https://github.com/ripple/validator-history-service/pull/32